### ISSUE TITLE
Accessibility / title support

### ISSUE THREAD

**Issue by benmccann** [sanniassin/react-input-mask#181]
Is it possible to add the `title` attribute to inputs?

Currently the examples fail accessibility standards, but adding a title would help substantially. Go to http://sanniassin.github.io/react-input-mask/demo.html in Chrome, open Dev tools by hitting F12, click Audits tab, run audits and see accessibility failures.

**Comment by benmccann** [comment]
Actually, I see that `title` does work and get passed through. That would be nice to add to the documentation as it wasn't initially clear to me.
